Core/Object Arts/Dolphin/IDE/Base/SmalltalkSearchPresenter.cls

```diff
@@ -154,47 +151,47 @@ rangeToCompleteAt: anInteger
 				ifFalse: [anInteger to: view caretPosition - 1]]!
 
 searchItemsStartingWith: aString maxItems: anInteger
```

Comment:
Can you note here (not in the code, I mean as a self-review comment) what the effect of the changes to this method are please. It's not easy to tell from inspection.